includes/admin/class-amp-post-meta-box.php

```diff
@@ -270,7 +270,7 @@ function () {
 			'errorMessages'   => $this->get_error_messages( $status_and_errors['errors'] ),
 			'hasThemeSupport' => ! amp_is_legacy(),
 			'isStandardMode'  => amp_is_canonical(),
-		];
+		] + self::get_featured_image_dimensions();
```

Comment:
This should rather use `array_merge()`.

Reply:
I have removed  `+` and replaced it with array_merge function